### PR TITLE
Fixes #133: Avoid running coverage twice.

### LINKED ISSUE #133: Avoid running coverage twice now that isparta is fixed
refs https://github.com/douglasduteil/isparta/pull/59

No more need to run the test suite twice to properly get coverage reporting.

### PR DESCRIPTION
**Pull request by n1k0**
refs #133.

### CODE REVIEW

**Comment by n1k0** [comment]
> All checks have passed
> 1 successful check 

Hold on merging, there's an issue.

**Comment by n1k0** [comment]
Seems this has been [broken for a while](https://coveralls.io/github/mozilla-services/kinto.js) actually.

**Comment by n1k0** [comment]
Problem seems to have occurred just after the project being renamed and moved to the mozilla-services org. Might be a coveralls.io issue.

**Comment by n1k0** [comment]
Indeed, coverage reporting is available [there](https://coveralls.io/github/Kinto/kinto.js). This is a strange problem, as we're no more notified to the right renamed repo.

**Comment by n1k0** [comment]
Coverage's back by tweaking Travis configuration it seems.

**Comment by n1k0** [comment]
r=? @leplatrem (welcome back :p)

**Comment by ametaireau** [comment]
LGTM.

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "publish-package": "npm run build && npm publish",
     "report-coverage": "npm run test-cover && ./node_modules/coveralls/bin/coveralls.js < ./coverage/lcov.info",
     "tdd": "mocha -w --compilers js:babel/register 'test/**/*_test.js'",
-    "test": "npm run lint && npm run test-nocover",
+    "test": "npm run lint && npm run report-coverage",
```
Comment (leplatrem) [inline package.json]: Running local tests with ``npm test`` will now make a call to coveralls ?

Comment (leplatrem) [inline package.json]: I meant: do we need to update the hacking docs accordingly ? :)

Comment (n1k0) [inline package.json]: > Running local tests with npm test will now make a call to coveralls ?

You're right we shouldn't. Maybe it's okay to run this twice after all. Closing.

**Pull request closed without merging**